dlc-manager/src/channel_updater.rs

```diff
@@ -507,7 +511,7 @@ where
             .fee_rate_per_vb,
     };
 
-    Ok((signed_channel, signed_contract))
+    Ok((signed_channel, signed_contract, fund_tx))
```

Comment:
nit: could you rename `fund_tx` to `signed_fund_tx`? I think it would make the code clearer